txclib/commands.py

```diff
@@ -75,17 +75,15 @@ def cmd_init(argv, path_to_tx):
         config.write(fh)
         fh.close()
 
-    prj = project.Project(path_to_tx)
-    prj.getset_host_credentials(transifex_host, username=options.user,
-                                password=options.password,
-                                token=options.token, force=options.save,
-                                no_interactive=options.no_interactive)
-    prj.save()
-
     if not options.skipsetup and not options.no_interactive:
         logger.info(messages.running_tx_set)
         cmd_config([], path_to_tx)
     else:
+        prj = project.Project(path_to_tx)
```

Comment:
@kouk it is not clear why setting the credentials should live inside the if statement, can you clarify this a bit? There are also tests that fail due to this change.

Reply:
because `cmd_config` in the first clause will do the same thing (so it happens twice). Before the change, when the password was saved to the file, the second time there was no prompt because it was already found. But since we're not saving it to the file with this change, the code under `cmd_config` will ask for the token again.